_includes/figure

```diff
@@ -27,10 +27,10 @@ because https://github.com/jekyll/jekyll/issues/2248.
 
 <div class="figure-body">
 
-<div class="figure-images contains-{{ number-of-images }}{% endif %}">
+<div class="figure-images contains-{{ number-of-images }}">
     {%for image in images %}
         {% if include.link %}<a href="{{ include.link }}">{% elsif site.output == "web" %}<a href="{{ path-to-book-directory }}/{{ site.image-set }}/{{ image }}">{% else %}{% endif %}
```

Comment:
@arthurattwell Isn't that last `{% else %}` still unnecessary since it's empty?
And might it be nicer to have the `if`s inside the `a`, like we do for the `alt` on the line below?

Then this line would be: (deep breath)

```
<a href="{% if include.link %}{{ include.link }}{% elsif site.output == "web" %}{{ path-to-book-directory }}/{{ site.image-set }}/{{ image }}{% endif %}">
```